src/components/ContentListTable/ContentListTable.tsx

```diff
@@ -210,11 +210,12 @@ const ContentListTable = () => {
             </Thead>
             <Tbody>
               {contentList.map(
-                ({ uuid, name, url, distribution_arch, distribution_versions }: ContentItem) => (
+                ({ uuid, name, url, distribution_arch, package_count, distribution_versions }: ContentItem) => (
```

Comment:
What value will you display if it is zero? 
"Unknown", 0, "N/A" ? 
The below is the screenshot, I think we may need design input here though. 
We'll likely want to have some N/A test and a hover popup that explains why.
![Screen Shot 2022-09-06 at 9 02 40 AM](https://user-images.githubusercontent.com/38083295/188669496-a4c9b287-f838-44b1-95f4-ecfca2afb4d3.png)

Reply:
For now maybe leave it as "0" and we can address what do here at a later time.